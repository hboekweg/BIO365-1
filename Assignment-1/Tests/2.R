source("2.R")
