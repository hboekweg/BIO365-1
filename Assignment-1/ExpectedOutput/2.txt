[1] 2
[1] 3
[1] "numeric"
[1] "numeric"
[1] 5
[1] "numeric"
[1] "integer"
[1] "numeric"
